lumen/variables/base.py

```diff
@@ -209,7 +209,7 @@ def __init__(self, **params):
         if 'value' not in params and default is not None:
             params['value'] = default
         params['name'] = self.label
-        deserialized = {}
+            params['name'] = self.label
```

Comment:
This doesn't look right.

Reply:
VSCode got crazy for a moment not staging the right changes.